README.md

```diff
@@ -110,16 +109,39 @@ export default {
 const state = reactive({
   list: [ref(0)],
 })
-// no unwrap, `.value` is required
-state.list[0].value === 0 // true
+state.list[0].value === 0
 
 state.list.push(ref(1))
-// no unwrap, `.value` is required
 state.list[1].value === 1 // true
 ```
 
 </details>
 
+<details>
+<summary>
+✅ <b>Should</b> always use <code>ref</code> in a <code>reactive</code> when working with <code>Array</code>
+</summary>
+
+```js
+const a = reactive({
+  list: [
+    reactive({
+      count: ref(0),
+    }),
+  ],
+})
+a.list[0].count === 0
+
+a.list.push(
+  reactive({
+    count: ref(1),
```

Comment:
Is the `ref` here really required? A bit too restricted imo.

Reply:
I just rolled back this file, currently there's a few issues when using reactive([]) better just tell to use ref on the readme